hbase_sql/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/ExternalTableIT.java

```diff
@@ -572,7 +562,7 @@ public void testWriteReadNullValues() throws Exception {
     @Test
     public void testWriteReadNullValuesAvro() throws Exception {
 
-        String tablePath = getExternalResourceDirectory()+"null_test_location_avro";
+        String tablePath = getTempOutputDirectory() + "null_test_location_avro";
```

Comment:
I think "null_test_location_avro" should be "/null_test_location_avro", otherwise, the temporary directory used would be something like "SpliceUnitTest_temp_xxxxnull_test_location_avro" instead of "SpliceUnitTest_temp_xxxxn/ull_test_location_avro". And the after test call deleteTempDirectory() wouldn't be able to delete this directory.  Everything else looks good to me.

Reply:
actually a lot of tests forgot the /, so i added it to _getTempOutputDirectory_

Reply:
done